packages/web/src/components/gcds-stepper/gcds-stepper.tsx

```diff
@@ -49,17 +54,21 @@ export class GcdsStepper {
   }
 
   render() {
-    const { currentStep, lang, totalSteps } = this;
+    const { currentStep, lang, totalSteps, tag } = this;
 
     return (
       <Host>
         <gcds-heading
-          tag="h6"
+          tag={tag}
           class="gcds-stepper"
           margin-top="0"
           margin-bottom="300"
         >
-          {`${i18n[lang].step} ${currentStep} ${i18n[lang].of} ${totalSteps}`}
+          <span class="gcds-stepper__steps">
+            {`${i18n[lang].step} ${currentStep} ${i18n[lang].of} ${totalSteps}`}
+            <gcds-sr-only> : </gcds-sr-only>
```

Comment:
Could you add a comment here with a brief explanation of the a11y benefit, please? :)